components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1237,6 +1242,12 @@ void AdsServiceImpl::TriggerNewTabPageAdEvent(
     const std::string& placement_id,
     const std::string& creative_instance_id,
     const ads::mojom::NewTabPageAdEventType event_type) {
+  if (!IsEnabled()) {
+    // Only record to P3A if user does not have ads enabled
```

Comment:
nit: comment is not needed due to `!IsEnabled`

Reply:
I wish to keep the comment so others can know why this condition exists. Expanded the comment to state that this is a requirement as per the spec.